config/locales/home.en.yml

```diff
@@ -14,6 +14,7 @@ en:
       popular_projects: 'Most Popular Projects'
       active_projects: 'Most Active Projects'
       active_contributors: 'Most Active Contributors'
+      recent_projects: 'Most Recent Projects'
```

Comment:
Let's change this to 'Recently Added Projects'.

Reply:
done